### ISSUE TITLE
DesktopGL projects can't be run on Linux anymore

### ISSUE THREAD

**Issue by s-m-k** [MonoGame/MonoGame#6341]
My game doesn't work on Linux anymore. The error can be reproduced with the basic MonoGame's DesktopGL template project.

The error log:
```
smk@smk-VirtualBox:~/Desktop/CleanMG$ mono Game1.exe 

Unhandled Exception:
System.TypeLoadException: Could not load type 'Microsoft.Xna.Framework.Audio.OggStreamer' from assembly 'MonoGame.Framework, Version=3.7.0.1549, Culture=neutral, PublicKeyToken=null'.
  at Microsoft.Xna.Framework.Audio.OpenALSoundController..ctor () [0x00016] in <0d619505e7934ce19a62ac4e7ca95c72>:0 
  at Microsoft.Xna.Framework.Audio.OpenALSoundController.get_GetInstance () [0x00007] in <0d619505e7934ce19a62ac4e7ca95c72>:0 
  at Microsoft.Xna.Framework.SdlGamePlatform..ctor (Microsoft.Xna.Framework.Game game) [0x000d2] in <0d619505e7934ce19a62ac4e7ca95c72>:0 
  at Microsoft.Xna.Framework.GamePlatform.PlatformCreate (Microsoft.Xna.Framework.Game game) [0x00000] in <0d619505e7934ce19a62ac4e7ca95c72>:0 
  at Microsoft.Xna.Framework.Game..ctor () [0x001fb] in <0d619505e7934ce19a62ac4e7ca95c72>:0 
  at Game1.Game1..ctor () [0x00000] in <d515b73056434b6cb5e8374540fdac5b>:0 
  at Game1.Program.Main () [0x00001] in <d515b73056434b6cb5e8374540fdac5b>:0 
[ERROR] FATAL UNHANDLED EXCEPTION: System.TypeLoadException: Could not load type 'Microsoft.Xna.Framework.Audio.OggStreamer' from assembly 'MonoGame.Framework, Version=3.7.0.1549, Culture=neutral, PublicKeyToken=null'.
  at Microsoft.Xna.Framework.Audio.OpenALSoundController..ctor () [0x00016] in <0d619505e7934ce19a62ac4e7ca95c72>:0 
  at Microsoft.Xna.Framework.Audio.OpenALSoundController.get_GetInstance () [0x00007] in <0d619505e7934ce19a62ac4e7ca95c72>:0 
  at Microsoft.Xna.Framework.SdlGamePlatform..ctor (Microsoft.Xna.Framework.Game game) [0x000d2] in <0d619505e7934ce19a62ac4e7ca95c72>:0 
  at Microsoft.Xna.Framework.GamePlatform.PlatformCreate (Microsoft.Xna.Framework.Game game) [0x00000] in <0d619505e7934ce19a62ac4e7ca95c72>:0 
  at Microsoft.Xna.Framework.Game..ctor () [0x001fb] in <0d619505e7934ce19a62ac4e7ca95c72>:0 
  at Game1.Game1..ctor () [0x00000] in <d515b73056434b6cb5e8374540fdac5b>:0 
  at Game1.Program.Main () [0x00001] in <d515b73056434b6cb5e8374540fdac5b>:0 
smk@smk-VirtualBox:~/Desktop/CleanMG$ 
```

The build otherwise works well on Windows and OSX. Tried both on VirtualBox and on a "real" installation.

#### How to reproduce:

1. Create an empty DesktopGL project.
2. Build it on Windows using .NET's tools
3. Run it using Mono runtime (`mono Game1.exe`) on a Ubuntu 18.04 machine.

#### What version of MonoGame does the bug occur on:
- MonoGame 3.7.0.1549

#### What operating system are you using:
- Linux / Ubuntu 18.04

#### What MonoGame platform are you using:

- DesktopGL

**Comment by tomspilman** [comment]
@cra0zy ?

**Comment by cra0zy** [comment]
I'll try to look into this tomorrow.

**Comment by cra0zy** [comment]
TLDR. @s-m-k was using the `mono-runtime` package without the side components, you usually want to use `mono-complete` if you are lazy when it comes to finding deps.

Packaging up the game properly fixes it so closing this.

**Comment by tomspilman** [comment]
Thanks @cra0zy !

@s-m-k - Everything good now?
